homepage/docker-compose.yaml

```diff
@@ -15,6 +15,7 @@ services:
       - HOMEPAGE_TEMPLATE_REPO
       - HOMEPAGE_TEMPLATE_REPO_SYNC_ON_START
       - HOMEPAGE_AUTO_CONFIG
+      - HOMEPAGE_ALLOWED_HOSTS=${HOMEPAGE_ALLOWED_HOSTS:-${HOMEPAGE_TRAEFIK_HOST}}
```

Comment:
While this default fallback works in bash i dont think it works in docker compose. Youd have to template it in instance.yaml